File: api-server/src/main/java/com/ggumtle/ggumtle/auth/presentation/response/LoginResponse.java
package com.ggumtle.ggumtle.auth.presentation.response;

import com.ggumtle.ggumtle.auth.application.result.LoginResult;

public record LoginResponse(
        Long memberId,
        String accessToken
) {
    public static LoginResponse from(LoginResult result){
        return new LoginResponse(
                result.memberId(),
                result.accessToken()
        );
    }
}

File: android/core/domain/src/main/java/com/example/domain/rest/model/auth/response/LoginResponse.kt
package com.example.domain.rest.model.auth.response

data class LoginResponse(
    val memberId: Long,
    val accessToken: String,
//    val refreshToken: String,
//    val email: String
) {
    fun isTokenValid(): Boolean = accessToken.isNotBlank() //&& refreshToken.isNotBlank()
}